### PR TITLE
Add expand as sigmoid api

### PR DESCRIPTION
**Pull request by phlrain**
### PR types
Others

### PR changes
OPs

### Describe
add expand as； sigmoid cross entropy

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/test_expand_as_v2_op.py [hunk 1]
```diff
@@ -21,78 +21,63 @@
 import paddle.fluid as fluid
 
 
-class TestExpandAsOpRank1(OpTest):
+class TestExpandAsBaisc(OpTest):
```
Comment (zyfncg) [inline python/paddle/fluid/tests/unittests/test_expand_as_v2_op.py]: Basic?

File: python/paddle/fluid/tests/unittests/test_sigmoid_cross_entropy_with_logits_op.py [hunk 2]
```diff
@@ -207,68 +219,71 @@ def setUp(self):
...
-                x2 = fluid.layers.data(
-                    name='x2', shape=[3, 4, 5, 6], dtype="int32")
-                lab2 = fluid.layers.data(
-                    name='lab2', shape=[3, 4, 5, 6], dtype="int32")
-                fluid.layers.sigmoid_cross_entropy_with_logits(x2, lab2)
-
-            self.assertRaises(TypeError, test_dtype)
+        self.check_grad(['X'], 'Out', check_eager=True)
+
+    class TestSigmoidCrossEntropyWithLogitsOp6(OpTest):
```
Comment (zyfncg) [inline python/paddle/fluid/tests/unittests/test_sigmoid_cross_entropy_with_logits_op.py:227]: 缩进好像有点问题？

File: python/paddle/fluid/tests/unittests/test_sigmoid_cross_entropy_with_logits_op.py [hunk 3]
```diff
@@ -207,68 +219,71 @@ def setUp(self):
...
+            term2 = (1 - self.inputs['Label']) * np.log(1 - sigmoid_X)
+            self.outputs = {'Out': -term1 - term2}
+
+        def test_check_output(self):
+            self.check_output(check_eager=True)
+
+        def test_check_grad(self):
+            self.check_grad(['X'], 'Out', check_eager=True)
+
+    class TestSigmoidCrossEntropyWithLogitsOpError(unittest.TestCase):
```
Comment (zyfncg) [inline python/paddle/fluid/tests/unittests/test_sigmoid_cross_entropy_with_logits_op.py:259]: 同上

**Review by chenwhql — DISMISSED** [review]
binary_cross_entropy_with_logits下面的好像没改，可能还得提一个PR
image

**Comment by phlrain** [comment]
> binary_cross_entropy_with_logits下面的好像没改，可能还得提一个PR image

已经修复

**Review by chenwhql — APPROVED** [review]
